contracts/erc20/contracts/src/LibERC20Token.sol

```diff
@@ -107,6 +108,50 @@ library LibERC20Token {
         }
     }
 
+    /// @dev Retrieves the allowance for a token, owner, and spender.
+    ///      Returns `0` if the call reverts.
+    /// @param token The address of the token contract.
+    /// @param owner The owner of the tokens.
+    /// @param spender The address the spender.
+    /// @return allowance The allowance for a token, owner, and spender.
+    function allowance(address token, address owner, address spender)
```

Comment:
Why do we return 0 instead of reverting if the call fails?

Reply:
This library is all about very permissive interactions. Plus I figured we really don't want to revert in DevUtils if we can help it.